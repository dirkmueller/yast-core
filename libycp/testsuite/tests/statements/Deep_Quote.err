[Scanner] tests/statements/Deep_Quote.ycp:17 Error: Undeclared identifier 'Time'
Parsed:
----------------------------------------------------------------------
YError
----------------------------------------------------------------------
[Interpreter] tests/statements/Deep_Quote.ycp:17 undeclared identifier
Parsed:
----------------------------------------------------------------------
{ return (8 + 5); }
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
{ return `Time; }
----------------------------------------------------------------------
[Scanner] tests/statements/Deep_Quote.ycp:20 Error: Undeclared identifier 'Time'
Parsed:
----------------------------------------------------------------------
YError
----------------------------------------------------------------------
[Interpreter] tests/statements/Deep_Quote.ycp:20 undeclared identifier
