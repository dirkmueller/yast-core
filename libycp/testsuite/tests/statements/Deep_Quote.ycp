
# ---------------------------------------------------------
#
#  Filename:    Deep_Quote.ycp
#
#  Purpose:     test cases for YCP test
#
#  Creator:     jd@suse.de
#
#  Maintainer:  jd@suse.de
#
# ---------------------------------------------------------


# --------- DEEP_QUOTE

( ``( Time ) )

