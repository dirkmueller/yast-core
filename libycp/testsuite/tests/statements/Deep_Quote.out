(nil)
((8 + 5))
(`Time)
(nil)
